src/app/zkapps_examples/test/tokens/tokens.ml

```diff
@@ -125,13 +125,12 @@ let%test_module "Tokens test" =
       let subtree =
         []
         |> Zkapp_command.Call_forest.cons
-             { Account_update.Poly.authorization =
-                 Control.Poly.Signature Signature.dummy
-             ; body =
-                 Zkapps_examples.mk_update_body (fst mint_to_keys)
-                   ~use_full_commitment:true ~balance_change:(int_to_amount 1)
-                   ~token_id:owned_token_id
-             }
+           @@ Account_update.with_aux
```

Comment:
TODO: worth making a simple follow-up PR which would define a function in the file which would take parameters `~may_use_token`, `~(balance_change : int)` and `~token_id`.

Otherwise I see a lot of code duplication.

Reply:
I can do that in a follow-up PR, sure.